css/src/atomics/typography.scss

```diff
@@ -1,3 +1,15 @@
+// Font stack
+
+.is-monospace {
+	font-family: $monospace-font-stack !important;
+	-moz-osx-font-smoothing: auto;
+	-webkit-font-smoothing: auto;
+}
+
+.is-quote {
```

Comment:
This item, at present at least, has very focused usefulness. I think we can remove it from the framework.